write/writer.go

```diff
@@ -15,6 +15,28 @@ import (
 	"github.com/suyashkumar/dicom/element"
 )
 
+// Option is the type used for options for writing DICOM elements
+type Option func(o *optSet)
+
+// OptSkipVRVerification disables VR verification when encoding elements
+var OptSkipVRVerification Option = func(o *optSet) {
```

Comment:
nit: no need to prefix with `Opt`, since it is of type `Option`
```suggestion
var SkipVRVerification Option = func(o *optSet) {
```